makefile

```diff
@@ -78,7 +78,7 @@ DOCKER_GOPATH_DIR = $(word 1,$(DOCKER_GOPATH_TOKS))
 # This is used to generate published artifacts; to raise the golang version we publish and run
 # tests-in-docker against, you have to update the tag of the image used here.
 GO_CMD = docker run --rm $(DOCKER_RUN_AS_USER) -v ${PWD}:/go/src/armada -w /go/src/armada $(DOCKER_NET) \
-	-e GOPROXY -e GOPRIVATE -e GOCACHE=/go/cache -e INTEGRATION_ENABLED=true -e CGO_ENABLED=0 -e GOOS=linux -e GARCH=amd64 \
+	-e GOPROXY -e GOPRIVATE -e GOCACHE=/go/cache -e INTEGRATION_ENABLED=true -e CGO_ENABLED=0 -e GOOS=darwin -e GARCH=arm64 \
```

Comment:
This is probably causing your integration test failures.  We probably should not change this since our CI is running on linux with amd.

Reply:
removed -- not necessary to specify arch now